neighsyncd/neighsyncd.cpp

```diff
@@ -29,10 +33,23 @@ int main(int argc, char **argv)
             netlink.dumpRequest(RTM_GETNEIGH);
 
             s.addSelectable(&netlink);
+            if (sync.getRestartAssist()->isWarmStartInProgress())
+            {
+                sync.getRestartAssist()->readTableToMap();
+                sync.getRestartAssist()->startReconcileTimer(s);
+            }
             while (true)
             {
                 Selectable *temps;
                 s.select(&temps);
+                if (sync.getRestartAssist()->isWarmStartInProgress())
+                {
+                    if (sync.getRestartAssist()->checkReconcileTimer(temps))
+                    {
+                        sync.getRestartAssist()->reconcile();
+                        sync.getRestartAssist()->stopReconcileTimer(s);
```

Comment:
>stopReconcileTimer [](start = 49, length = 18)

First stopReconcileTimer, then reconcile?

Reply:
It should not matter as there was a warmstartinprogress flag check before, but it is fine to move the other way around.